src/abstractdatatable/io.jl

```diff
@@ -44,17 +44,18 @@ function printtable(io::IO,
         for j in 1:p
             if !isnull(dt[j][i])
                 if ! (etypes[j] <: Real)
-		    print(io, quotemark)
-		    escapedprint(io, get(dt[i, j]), quotestr)
-		    print(io, quotemark)
+                    print(io, quotemark)
+                    x = isa(dt[i, j], Nullable) ? unsafe_get(dt[i, j]) : dt[i, j]
```

Comment:
`isa(dt[i, j], Nullable)` is not needed, `unsafe_get` works for non-`Nullable` too. Same below.